src/app/features/collectibles/stacks-non-fungible-tokens.tsx

```diff
@@ -0,0 +1,65 @@
+import StacksNftBns from '@assets/images/stacks-nft-bns.png';
+import StacksNft from '@assets/images/stacks-nft.png';
+import { Box } from '@stacks/ui';
+
+import type { StacksNonFungibleTokenAssetBalance } from '@shared/models/crypto-asset-balance.model';
+
+import { useCurrentAccountDisplayName } from '@app/common/hooks/account/use-account-names';
+
+import { BaseCollectible } from './components/base-collectible';
+
+interface StacksNftCryptoAssetsProps {
+  assetBalances: StacksNonFungibleTokenAssetBalance[];
+}
+export function StacksNonFungibleTokens({ assetBalances }: StacksNftCryptoAssetsProps) {
```

Comment:
Lots of duplicate code, layout component would be great + separate bns/nft components if they're that different.

Reply:
I had setup a `crypto-assets` folder under `components` where you can see I moved the list from. I had a `StacksNonFungibleTokenAssetItem` and a layout component setup as a placeholder for this work. But, was rolling with `Collectibles` now that Edu set it up.